feature/office/src/main/java/com/iteneum/office/presentation/ui/OfficeView.kt

```diff
@@ -50,55 +77,51 @@ fun OfficeView() {
- * @author Andres Ivan Medina
+ * @modifiedBy Andres Ivan Medina
  */
-@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun OfficeUI(
-    officeData: OfficeModel,
+    officeData: Office,
     onCallButtonClicked: () -> Unit,
     onEmailButtonClicked: () -> Unit,
 ) {
-    Scaffold(content = { paddingValues ->
+    Column(
+        modifier = Modifier.fillMaxSize()
+    ) {
         val sizes = LeasePertTheme.sizes
-        Box(modifier = Modifier.padding(paddingValues)) {
-            Column(
-                Modifier.fillMaxSize()
-                //.padding(top = sizes.extraSize18)
-            ) {
-                LPTitleLarge(label = stringResource(
-                    id = R.string.LPInfo
-                ),
-                    color = MaterialTheme.colorScheme.tertiary,
-                    textAlign = TextAlign.Justify,
-                    weight = FontWeight.Bold,
-                    onClick = {})
-                LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
-                    label = officeData.address,
-                    color = MaterialTheme.colorScheme.onPrimaryContainer,
-                    textAlign = TextAlign.Justify,
-                    weight = FontWeight.Normal,
-                    onClick = {})
-                LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
-                    label = stringResource(id = R.string.LPHours) + officeData.schedule,
-                    color = MaterialTheme.colorScheme.onPrimaryContainer,
-                    textAlign = TextAlign.Justify,
-                    weight = FontWeight.Normal,
-                    onClick = {})
-                LpOutlinedButton(Modifier.padding(top = sizes.extraSize10),
-                    icon = Icons.Filled.Call,
-                    textButton = stringResource(id = R.string.LPCallButton),
-                    onClicked = {
-                        onCallButtonClicked()
-                    })
-                LpOutlinedButton(Modifier.padding(top = sizes.extraSize10),
-                    icon = Icons.Outlined.Mail,
-                    textButton = stringResource(id = R.string.LPMailButton),
-                    onClicked = {
-                        onEmailButtonClicked()
-                    })
+        LPTitleLarge(label = stringResource(
+            id = R.string.LPInfo
+        ),
+            color = MaterialTheme.colorScheme.tertiary,
+            textAlign = TextAlign.Justify,
+            weight = FontWeight.Bold,
+            onClick = {})
+        LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
+            label = officeData.address,
+            color = MaterialTheme.colorScheme.onPrimaryContainer,
+            textAlign = TextAlign.Justify,
+            weight = FontWeight.Normal,
+            onClick = {})
+        LPBodyMedium(modifier = Modifier.padding(top = sizes.extraSize10),
+            label = stringResource(id = R.string.LPHours) + officeData.schedule,
```

Comment:
use stringResource() and pass **officeData.schedule** as parameter

Reply:
Hours are supose to be a whole string?  or just the hours like in:

Hours: [  9am to 6pm ]

Reply:
changed all the string into the object response